### ISSUE TITLE
what is the purpose of CENTERNET.IGNORE_HIGH_FP ?

### ISSUE THREAD

**Issue by Anymake** [xingyizhou/CenterNet2#40]
what is the purpose of CENTERNET.IGNORE_HIGH_FP ? 
In the heatmap_focal_loss code, it seems to ignore the loss of high FP ? Isn't the loss of high FP more important?
`    if ignore_high_fp > 0:
        not_high_fp = (pred < ignore_high_fp).float()
        neg_loss = not_high_fp * neg_loss`

**Comment by xingyizhou** [comment]
Hi,
  Great question! This is mostly designed for LVIS, where the objects are not fully annotated, and the high FPs are sometimes true objects. It makes an ~0.5 mAP difference in LVIS (if I remembered correctly). For COCO, removing it is totally fine.
